unlock-app/src/__tests__/components/interface/account-settings/ChangePassword.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import * as rtl from 'react-testing-library'
+import {
+  ChangePassword,
+  validatePassword,
```

Comment:
is there anywhere else validatePassword would be used? If so, it could go in `utils/`

Reply:
I would suggest opening up an issue to refactor this in a future PR, not in this one.

Reply:
Yup, that's something I had in mind to do as part of the broader password refactor (since the current implementation is about as rudimentary as possible).